pilot/pkg/config/kube/gateway/testdata/zero.yaml.golden

```diff
@@ -40,7 +38,6 @@ apiVersion: networking.istio.io/v1
 kind: VirtualService
 metadata:
   annotations:
-    internal.istio.io/parents: HTTPRoute/http.default
```

Comment:
While we don't _need_ this now, might be nice to keep for debugging?

Reply:
especially when we merge them

Reply:
+1 to keeping